server/UserRegistration.java

```diff
@@ -0,0 +1,357 @@
+    }
+
+    public static String getNewUserAccountNumber() {
+        int min = 11121211;
+        int max = 99879199;
+        int accountNumber = (int) Math.floor(Math.random() * (max - min + 1) + min);
+        System.out.println(" ");
+        System.out.println("\n The new account the account number and sortcode will be:");
+        System.out.println("                 Account Number:" + " " + accountNumber);
+        return String.valueOf(accountNumber);
+    }
+
+    public static String getNewUserSortCode() {
+        int min = 111212;
+        int max = 998791;
+        int accountSortCode = (int) Math.floor(Math.random() * (max - min + 1) + min);
+        System.out.println("                 Sortcode:" + " " + accountSortCode);
+        return String.valueOf(accountSortCode);
+    }
+
+
+    public static String getNewUserInputNewPassword() {
+        //Boolean password conditions = to check that:
+        // Contains more than 8 chars.
+        // Contains at least one numerical value.
+        // Contains at least one capital letter.
+        // Contains at least one lowercase letter.
+
+        Scanner sc = new Scanner(System.in);
+        String newPassword;
+        boolean passwordChecks;
+        String userAnswerPassword;
+        do {
+            passwordChecks = true;
+            System.out.println(" ");
+            System.out.println("\nYou will now need to create a password for this account");
+            System.out.print("\nThe password must be (At least 8 letters long, contain a capital letter, a lowercase letter and numerical value)");
+            System.out.println("\nPlease create your new password: ");
+            newPassword = sc.nextLine();
+            if (!passwordLength(newPassword)) {
+                passwordChecks = false;
+            }
+            if (!passwordChecks(newPassword)) {
+                passwordChecks = false;
+            }
+        } while (!passwordChecks);
+        do {
+            System.out.println("\n Your new password is:" + " " + newPassword);
+            System.out.println("\nConfirm password (Y/N): ");
+            userAnswerPassword = sc.nextLine().toUpperCase();
+            if (userAnswerPassword.equals("N")) {
+                System.out.println("\nThe password must be (At least 8 letters long, contain a capital letter, a lowercase letter and numerical value)");
+                System.out.println("\nPlease create your new password: ");
+                newPassword = sc.nextLine();
+                System.out.println("Ok the password has now been updated:" + newPassword);
+                System.out.println("\nConfirm new password(Y/N): ");
+                userAnswerPassword = sc.nextLine().toUpperCase();
+            }
+        } while (!passwordChecks & userAnswerPassword.equals("N"));
+
+        System.out.println("\n Your new password is:" + " " + newPassword);
+        return newPassword;
+    }
+
+
+    public static boolean passwordLength(String newPassword) {
```

Comment:
The function is called `passwordLength` but it actually calls another function, `passwordChecks` if the length is valid (> 7). 
This is not an ideal function design, because the function does more things than it should (ideally 1, when possible), but if it already does, it should be named properly, for example `doesPasswordFulfillChecks`.
You don't need to change it now, but it's good to think about when designing functions.